pnpm-wireit/package-scripts.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * We're only using `nps` as a proxy for scripts in root `package.json` to
+ * make them available within a workspace.
+ */
+const { scripts } = require("./package.json");
```

Comment:
Using `nps` is a good way to keep scripts DRY, but keeping those scripts in `root:package.json` is pretty confusing.  

It took me a while to piece together the build chain where `root:build` calls `packages:build` calls `nps pkg:build:esm` calls `root:pkg:build:esm` ... it's kinda confusing why `root` is in there twice.
I actually think it might be slightly easier if we used `package-scripts.js`, so that: 
- the root `package.json` is for workspace-wide commands
- each `packages/*/package.json` is for defining the `wireit` config, but no build scripts
- the root `package-scripts.json` is for the common build scripts

Reply:
I think that's fair for keeping scripts in `package-scripts.js`. Don't really mind either way.

Reply:
@scottrippey In https://github.com/FormidableLabs/monorepo-infra-experiments/pull/6/commits/7ca9056e4b8b65bd778309d677693929c9e8414f I updated things with a note that:

1. root tasks go in package.json
2. workspace tasks go in package-scripts.js

Does that seem like a better system to you?

Reply:
Yes I love the change.  Looks great.  Thanks for the explanatory note too.